receiver/azureeventhubreceiver/README.md

```diff
@@ -36,6 +45,48 @@ receivers:
 
 This component can persist its state using the [storage extension].
 
+## Encoding
+
+### raw
+
+The "raw" encoding maps the AMQP properties and data into the
+attributes and body of an OpenTelemetry LogRecord, respectively.
+The body is represented as a raw byte array.
+
+### data
```

Comment:
Is data descriptive enough? Can we use "azurelogformat" or similar?

Reply:
Used "format" for the parameter and "raw" and "azure" for the constants.